sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1,3090 +1,3131 @@
-              "InvalidBlobState",
-              "BackupTimedOut",
-              "AzureStorageTimeout",
-              "GenericTimeout",
-              "InvalidThrottleParameter",
-              "EventHubLinkAlreadyClosed",
-              "ReliableBlobStoreError",
-              "RetryAttemptsExhausted",
-              "AzureTableStoreError",
-              "CheckpointStoreNotFound",
-              "DocumentDbInvalidReturnValue",
-              "ReliableDocDbStoreStoreError",
-              "ReliableBlobStoreTimeoutError",
-              "ConfigReadFailed",
-              "InvalidContainerReceiveLink",
-              "InvalidPartitionEpoch",
-              "RestoreTimedOut",
-              "StreamReservationFailure",
-              "SerializationError",
-              "UnexpectedPropertyValue",
-              "OrchestrationOperationFailed",
-              "ModelRepoEndpointError",
-              "ResolutionError",
-              "UnableToFetchCredentials",
-              "UnableToFetchTenantInfo",
-              "UnableToShareIdentity",
-              "UnableToExpandDiscoveryInfo",
-              "UnableToExpandComponentInfo",
-              "UnableToCompressDiscoveryInfo",
-              "UnableToCompressComponentInfo",
-              "GenericBadGateway",
-              "InvalidResponseWhileProxying",
-              "GenericServiceUnavailable",
-              "ServiceUnavailable",
-              "PartitionNotFound",
-              "IotHubActivationFailed",
-              "ServerBusy",
-              "IotHubRestoring",
-              "ReceiveLinkOpensThrottled",
-              "ConnectionUnavailable",
-              "DeviceUnavailable",
-              "ConfigurationNotAvailable",
-              "GroupNotAvailable",
-              "HostingServiceNotAvailable",
-              "GenericGatewayTimeout",
-              "GatewayTimeout"
-            ],
-            "type": "string"
-          },
-          "errorStatus": {
-            "description": "Additional details associated with the error.",
-            "type": "string"
-          },
-          "moduleId": {
-            "type": "string"
-          },
-          "operation": {
-            "type": "string"
-          }
-        }
-      },
-      "DeviceRegistryOperationWarning": {
-        "description": "Encapsulates device registry operation error details.",
-        "type": "object",
+        },
         "properties": {
-          "deviceId": {
-            "description": "The ID of the device that indicated the warning.",
-            "type": "string"
-          },
-          "warningCode": {
-            "enum": [
-              "DeviceRegisteredWithoutTwin"
-            ],
-            "type": "string"
-          },
-          "warningStatus": {
-            "description": "Additional details associated with the warning.",
-            "type": "string"
+          "$ref": "#/definitions/PropertyContainer",
+          "description": "Properties are optional and defaults to empty object."
```

Comment:
This doesn't really say what a `PropertyContainer` is. We should ask for clarification on this from service.

Reply:
I'll mark this as a TODO instead of using this description, sure